Makefile

```
@@ -0,0 +1,40 @@
+.PHONY: build test clean install build-all
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)